renku/cli/_format.py

```diff
@@ -17,10 +17,12 @@
 # limitations under the License.
 """Serializers for graph data."""
 
+import functools
+
 import click
 
 
-def ascii(graph):
+def ascii(graph, **kwargs):
```

Comment:
I would remove the `**kwargs` since we don't have generic options that would apply to all formatters.

Reply:
@rokroskar do we still need `**kwargs` everywhere?

Reply:
oops no, I forgot to remove them there.